springwolf-core/src/test/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/AsyncListenerAnnotationScannerTest.java

```diff
@@ -168,6 +171,41 @@ void scan_componentHasMultipleListenerAnnotations() {
                                 "test-channel-2", expectedChannel2));
     }
 
+    @Test
+    void scan_componentHasAsyncMethodAnnotation() {
```

Comment:
can we add this test also to the AsyncPublisherAnnotation?

Reply:
Done